src/main/java/hng_java_boilerplate/product/entity/Product.java

```diff
@@ -4,53 +4,29 @@
 import hng_java_boilerplate.user.entity.User;
 import jakarta.persistence.*;
 import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 
 @Entity
 @NoArgsConstructor
 @AllArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+@Data
 @Table(name = "products")
 public class Product {
     @Id
     private String id;
 
     private String name;
     private String description;
+    private String category;
+    private double price;
+    @Column(name = "image_url")
+    private String imageUrl;
 
     @ManyToOne
     @JsonIgnore
     @JoinColumn(name = "user_id", nullable = false)
     private User user;
-
-    public String getId() {
```

Comment:
Why did you remove all the setters?

Reply:
The @Data annotation provides the getters and setters